third_party/pcsc-lite/naclport/server/src/google_smart_card_pcsc_lite_server/global.cc

```diff
@@ -68,9 +71,35 @@ void PcscLiteServerDaemonThreadMain() {
   }
 }
 
+PcscLiteServerGlobal* g_pcsc_lite_server = nullptr;
+
+const char kReaderInitAddMessageType[] = "reader_init_add";
```

Comment:
nit: Either move this constants block next to the kLoggingPrefix constant definition, or move the kLoggingPrefix constant definition here.